src/Test/Utilities/Portable/CommonTestBase.cs

```diff
@@ -539,10 +539,10 @@ internal static void VerifyParentOperations(SemanticModel model)
                 // check SearchparentOperation return same parent
                 Assert.Equal(((Operation)child).SearchParentOperation(), parent);
 
-                if (parent == null && child.Kind != OperationKind.None)
```

Comment:
now we don't need any special case for GetOperation test since it no longer requires those. it works in all cases.